### PR TITLE
Update First Android Submission FYI guide

### PR DESCRIPTION
**Pull request by amandeepmittal**
## Why

Fixes ENG-12964

## How
By updating screenshots and instructions to create the first manual submission on Google Play Console.

Also added a section about further reading with essential resources.

### CODE REVIEW

**Review by keith-kurak — APPROVED** [review]
Thank you for updating this! As we mentioned a little disclaimer about Google changing stuff would be good :)

File: first-android-submission.md [hunk 1]
```diff
@@ -1,56 +1,76 @@
...
 
-#### 🤔 What Happened
+If you are submitting your Android app to Google Play Store **for the first time**, Google requires you to upload it manually through Google Play Console &mdash; a web UI. This guide provides all the steps you need to follow to manually upload your app to Play Console and create your first release.
 
-If you're submitting your Android app to Google Play Store **for the first time**, Google requires that you manually upload it through their web user interface. After you have submitted it once through the web user interface, you will be able to use `eas submit -p android` for subsequent submissions.
+> [!NOTE]
+> After you have submitted your app manually through Play Console, you will be able to use `eas submit -p android` for subsequent submissions.
 
-#### 💡 Solution
+> [!DISCLAIMER]
```
Comment (amandeepmittal) [inline first-android-submission.md:8]:
```suggestion
> [!WARNING]
```

File: first-android-submission.md [hunk 2]
```diff
@@ -1,56 +1,76 @@
...
+> Eventually, Google Play Console dashboard gets updated and to keep track of those changes is out of our scope. If you find something from this guide does not align with the Play Console, open an issue to let us know.
```
Comment (amandeepmittal) [inline first-android-submission.md:9]:
```suggestion
> Google Play Console dashboard gets updated, and keeping track of those changes is out of our scope. If you find something from this guide does not align with the Play Console, open an issue to let us know.
```

**Comment by minegamer32256-wq** [comment]
https://github.com/user-attachments/assets/09492769-60cc-42c2-abb2-871171142c98

**Pull request merged**